shop-service/product-service/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "product-service",
+  "version": "1.0.0",
+  "description": "Product service",
+  "main": "handler.ts",
+  "scripts": {
+    "test": "jest"
```

Comment:
I would recommend to add serverless related commands into package.json as well

Reply:
done